meta/SaiSerialize.cpp

```diff
@@ -2409,6 +2482,7 @@ static void sai_deserialize_qos_map_params(
     params.prio           = j["prio"];
     params.pg             = j["pg"];
     params.queue_index    = j["qidx"];
+    params.fc             = j["fc"];
```

Comment:
to be backward compatible you need to check if fc field exists, similar to field mpls_exp below

Reply:
then you probably dont need to modify dumpX.json files

Reply:
Thanks, need to revert my change to dumpX.json files as that is causing more problems.

Reply:
Done